launch/percep.launch

```diff
@@ -25,12 +25,41 @@
     -urdf -param robot_description
     -model rover" />
 
-  <!-- Perception-->
-  <node name="aruco_detect" pkg="mrover" type="aruco_detect">
-    <param name="publish_images" value="true" />
-    <param name="fiducial_len" value="0.20" />
-    <param name="dictionary" value="0" /> <!-- DICT_4X4_50 -->
-    <remap from="camera" to="camera/color/image_raw" />
-    <remap from="camera_info" to="camera/color/camera_info" />
+  <!--
+    ==========
+    Perception
+    ==========
+  -->
+<!--   <node name="aruco_detect" pkg="mrover" type="aruco_detect" /> -->
+
+  <!--
+    ============
+    Localization
+    ============
+    Frames in the free tree:
+    odom: The topmost global frame
+    base_link: The rover
+  -->
+  <node name="mrover_localization" pkg="mrover" type="localization.py" />
+
+  <rosparam command="load" file="$(find mrover)/params/ekf_localization.yaml" />
+
+  <!-- Fuses together GPS position and IMU orientation with an Extended Kalman Filter (EKF) -->
+  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_odom" clear_params="true">
+    <!-- Publications -->
+    <remap from="odometry/filtered" to="odometry/filtered" />
```

Comment:
Might need to go over this whole system another time

Reply:
> still confused by this.....

you mean how the remap is unnecessary or how this thing works?

Reply:
@qhdwight I actually don't think we should enforce this since it's a confusing misuse of remap. imo if we want documentation of what topics different nodes publish/subscribe to outside of our system diagram, then we should just require comment blocks above nodes in launch files like ```
<!-- publishes to:
        /topic/a
        /topic/b
        ...
    
     subscribes to:
        /topic/c
        /topic/d -->
```
this is exactly the same information that `rosnode info <node>` already provides, but I think it's reasonable to comment it in the launch file

Reply:
oh yeah don't do the remap if its just to annotate the channel names in there. Honestly the RQT graph should show all of this but for some reason it was fucked up last time I fired it up. I will investigate that again.

Reply:
Can't say I agree on the comments, I think just leaving in the remap for everything is better since it avoids duplication of information.